internal/controller/cluster_controller.go

```diff
@@ -928,6 +929,23 @@ func (r *ClusterReconciler) reconcilePods(
 		return ctrl.Result{RequeueAfter: 1 * time.Second}, ErrNextLoop
 	}
 
+	// Stop acting here if the Pods don't have received
+	// the latest PostgreSQL configuration.
```

Comment:
Actually I think this whole comment is stale. The following code paragraph clarifies we request (stop acting) if there are pods in different configs